lcls-twincat-common-components/Library/Devices/PPM/FB_PPM.TcPOU

```diff
@@ -163,7 +166,8 @@ stYag2 := astPositionState[E_PPM_States.YAG2];
 fbPowerMeter();
 fbGige();
 fbFlowMeter(fOffset:=fFlowOffset);
-fbYagThermoCouple();]]></ST>
+fbYagThermoCouple();
+fbFlowSwitch();]]></ST>
```

Comment:
The implementation is good.
What's the context here? The PPM can have either a flow switch, a flow meter, or both? The previous fbFlowMeter is still present, so I'm probably just out of the loop. Do we need to have some sort of mechanism to decide which to show in the UI later?

Reply:
Or, should we be doing something in the PLCs to handle this split?

Reply:
Yea, the former. The PPMs in the EBD/FEE have flow switches and the hutches have flow meters. We will need to deal with this somewhere; I thought on the GUI level, but open to suggestions.

Reply:
I think I agree, we can make a sequel to PPMCool